### DISCUSSION TITLE
dragging imgui outside of main window (multi viewports) on d3d11

### DISCUSSION THREAD

**Discussion by jakehvh** [ocornut/imgui#7548] [New Users Build/Link/Run issues ONLY!]
hello, how can i enable/use multi viewports on the latest version of imgui? i couldnt find anything about it anywhere

**Comment by nicolasnoble** [comment]
It's still part of [the docking branch](https://github.com/ocornut/imgui/tree/docking), with [its examples](https://github.com/ocornut/imgui/blob/docking/examples/example_glfw_opengl3/main.cpp#L81) demonstrating its multi viewports feature.

**Comment by ocornut** [comment]
See https://github.com/ocornut/imgui/wiki/Multi-Viewports#how-can-i-enable-multi-viewports-
